subprojects/android-test/android-log/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+    `maven-publish`
+    id("com.jfrog.bintray")
+}
+
+dependencies {
+    api(project(":common:logger"))
+
+    implementation(project(":common:sentry"))
+}
```

Comment:
This module doesn't depend on test libraries.
How about introducing `subprojects/android` libraries to highlight such type of libraries?

Reply:
`android-lib` already existed for this purpose, but this logger doesn't meant to be used anywhere but in tests for now, so i intentionally put it there